common/djangoapps/track/views/tests/test_views.py

```diff
@@ -138,13 +146,17 @@ def test_server_track(self):
             'event': '{}',
             'agent': '',
             'page': None,
-            'time': expected_time,
+            'time': FROZEN_TIME,
             'host': 'testserver',
             'context': {},
         }
-        self.mock_tracker.send.assert_called_once_with(expected_event)
+        self.assert_mock_tracker_call_matches(expected_event)
+
+    def assert_mock_tracker_call_matches(self, expected_event):
+        self.assertEqual(len(self.mock_tracker.send.mock_calls), 1)
+        actual_event = self.mock_tracker.send.mock_calls[0][1][0]
```

Comment:
Yeah not really... however, this is "zombie" code since I want to gut server_track and replace all of this with the cleaner pattern now used for user_track testing.